packages/client/wallets/aa/src/blockchain/wallets/FireblocksNCWallet.ts

```diff
@@ -61,7 +61,10 @@ export const FireblocksNCWallet = async (
         return crossmintService.getNCWIdentifier(deviceId);
     });
 
+    const testnets: Blockchain[] = [BlockchainTestNet.GOERLI, BlockchainTestNet.MUMBAI, BlockchainTestNet.SEPOLIA, BlockchainTestNet.TESTNET]
+    const testnet = testnets.includes(chain)
```

Comment:
@mPaella I assume there's a better TS way to do this, but I was unable to treat `BlockchainTestNet` as a dict to do something like `.values().includes(...)`, I was only able to treat it as a type